### PR TITLE
HomeWork5

### PR DESCRIPTION
**Pull request by MatveiNikita12**

### CODE REVIEW

File: src/main/java/homework_nr_5/Months.java
Comment (gh3rtzz) [inline src/main/java/homework_nr_5/Months.java:1]: Enums should be all CAPS

Comment (MatveiNikita12) [reply src/main/java/homework_nr_5/Months.java:1]: Перевел

Comment (gh3rtzz) [reply src/main/java/homework_nr_5/Months.java:1]: Months enums are not in CAPS

Comment (MatveiNikita12) [reply src/main/java/homework_nr_5/Months.java:1]: поменял

**Review by gh3rtzz — APPROVED** [review]
approved

**Pull request merged**